### ISSUE TITLE
This plugin #excalidraw takes too long to load, affecting the application startup time and potentially causing other plugins to fail to load.

### ISSUE THREAD

**Issue by eight-nines** [haydenull/logseq-plugin-excalidraw#25]
### Search first

- [X] I searched and no similar issues were found

### What Happened?

The plugin was working fine before, but today when I opened logseq, the following error appeared, I tried changing the version of logseq, and changing the version of the plugin, neither of which took effect

This plugin #excalidraw takes too long to load, affecting the application startup time and potentially causing other plugins to fail to load.

### Reproduce the Bug

The plugin was working fine before, but today when I opened logseq, the following error appeared, I tried changing the version of logseq, and changing the version of the plugin, neither of which took effect

This plugin #excalidraw takes too long to load, affecting the application startup time and potentially causing other plugins to fail to load.

### Are you willing to submit a PR? If you know how to fix the bug.

- [ ] I'm willing to submit a PR (Thank you!)

**Comment by haydenull** [comment]
May I ask which version of Logseq and plugins you are using?

Currently, the cause of this issue has not been identified. My another plugin occasionally displays this prompt as well, but it dose not affect its functionality. Have you encountered any issues at the moment?

**Comment by eight-nines** [comment]
Logseq 9.*/0.10.1   &&  plugins all version
Plugin not available

**Comment by haydenull** [comment]
You can try creating a new logseq graph, maybe the data of a certain drawing is incorrect.

Furthermore, when you mention that the plugin is not available, are you referring to the toolbar icon not being displayed or the drawing feature not appearing in the notes?

**Comment by eight-nines** [comment]
I don't think it has anything to do with "the data of a certain drawing is incorrect", today I restarted my computer and the plugin suddenly worked, I restarted it a couple times before but no change.
yes， the toolbar icon not being displayed and the drawing feature not appearing in the notes.

**Comment by haydenull** [comment]
Based on the current information, I am unable to determine whether it is a plugin issue or a Logseq issue. Let’s observe and see if there is any way to reproduce it in the future.

**Comment by manuelspezzani** [comment]
Today I've got the same notification. 

- Logseq 0.10.9
- Excalidraw 1.5.1

Unfortunately I'm unable to reproduce the issue: restarting Logseq apparently fixed it.
